src/routes/+layout.svelte

```diff
@@ -206,7 +208,9 @@
 </svelte:head>
 
 {#if loaded}
+	<IonosHeader />
```

Comment:
lets put our components outside of if loaded in order to show it due loading process
```svelte
<IonosHeader />
{#if loaded}
	<slot />
{/if}
<IonosFooter />
```